test/scripts/e2e.sh

```diff
@@ -53,15 +62,19 @@ reset_dirs
 echo Killing all instances and installing current build
 
 pkill -u $(whoami) -x algod || true
-./scripts/local_install.sh -c ${CHANNEL} -p ${BINDIR} -d ${DATADIR}
 
+if [[ ! ${NO_BUILD} == "True" ]]; then
```

Comment:
Same as my comment above, you can use a boolean here rather then checking a string.

Reply:
OK, will change to use booleans.